revision3_2581/paper.tex

```diff
@@ -337,7 +336,7 @@
 \end{CodeInput}
 \end{CodeChunk}
 %
-\pkg{webchem} always returns a named list (one entry for each substance) and the available information content can be very voluminous.
+When possible, \pkg{webchem} returns a data frame with one row per substance.  However, data from some sources can be very voluminous and not tabular. In these cases \pkg{webchem} always returns a named list (one entry for each substance).
```

Comment:
Much better, thanks! For `match = "all"` the data frame may contain more than one row per substrate. How about: "... a data frame with one or more rows per substrate."?

Reply:
Sure, although I think it's fine the way it is.  You might get multiple rows per *query* (because the query matches multiple substances), but ideally we would aim for one row per substance.